src/SapiDispatcher.php

```diff
@@ -5,6 +5,8 @@
 use Drupal\Core\Config\ConfigManager;
 use Drupal\Core\Session\AccountProxy;
 use Drupal\Core\Routing\CurrentRouteMatch;
+use Drupal\Component\Plugin\PluginManagerInterface;
+use Drupal\sapi\StatisticsItemInterface;
```

Comment:
@maijs is the full StatisticsItemInterface use statement necessary when the two elements are in the same namespace?